packages/client/src/components/ModuleTree/Module/Module.tsx

```diff
@@ -1,25 +1,104 @@
 import React from 'react';
-import { Module as EModule } from '@codement/api';
-import classnames from 'classnames';
-import { Icon } from '@codement/ui';
-import Hex from './hex.svg';
-import styles from './Module.module.css';
+// import classnames from 'classnames';
+
+import { Module as EModule, UserModule, UserConcept } from '@codement/api';
+import { theme } from '@codement/ui';
+
+import { ModuleIcon } from './Icons/Icon';
 
 export interface ModuleProps {
-  module: EModule
+  module: EModule,
+  myModule?: UserModule,
+  myConcepts?: UserConcept[]
 }
 
 export const Module: React.FC<ModuleProps> = ({
-  module
-}) => <div className={styles.module}>
-  <div className="relative">
-    <Hex
-      className={classnames('h-12 stroke-2', {
-        // TODO: Add completed to module
-        completed: false
-      })}
-    />
-    <Icon icon="fire" size={4} />
-  </div>
-  <span className="text-xs">{module.name}</span>
-</div>;
+  module,
+  myModule,
+  myConcepts
+}) => {
+  const isCompleted = myModule?.completedAt;
+  const isOpen = myModule && !myModule.completedAt;
+  const isLocked = !myModule;
+
+  return <div style={{
+    textAlign: 'center',
+    marginBottom: '1.5rem',
+    color: theme.colors.primary['500'] }}
+  >
+
+    <div style={{
```

Comment:
I think we shouldn't use inline styles and use either styled components (first option) or class names (second)

Reply:
changed to styled-component. 
commit: fix: change inline style to styled-components 31d9a1e